src/lib/moduleMocker.js

```diff
@@ -47,6 +47,48 @@ function getType(ref) {
   return null;
 }
 
+function PlainConstructor() {}
+var plainObject = new PlainConstructor();
+/**
+ * methods on ES6 classes are not enumerable, so they can't
+ * be found with a simple `for (var slot in ...) {`
+ *
+ * scan will return an object with the same shape you would
+ * get with `for ... in` except the non-enumerable methods
+ * are included.
+ */
+ function scan(obj) {
+   var fakeObj = {};
+   var prop;
+   for (prop in obj) {
+     fakeObj[prop] = true;
+   }
+   if (obj) {
+     var props = [];
+     var tmp = obj;
+     var descriptor;
+
+     do {
+       props = Object.getOwnPropertyNames(tmp);
+
+       for (var i = 0; i < props.length; i++) {
+         prop = props[i];
+         // Skip methods that exist on all objects.
+         if (!plainObject[prop]) {
```

Comment:
`tmp !== Object.prototype` isn't going to work, it won't filter anything out. The purpose of this is to make sure we don't mock things like `constructor`. If you do mock `constructor` then everything blows up.

Maybe you meant `tmp[prop] != Object.prototype[prop]`, which could work but I don't see the point.